File: numerical-palindrome-number-1/numerical-palindrome-number-1.py
def palindrome(num):
    if isinstance(num,str) or num<0:
        return 'Not valid'
    num = str(num)
    l = len(num) 
    return num[0:l//2]==num[l//2:][::-1] if l%2==0 else num[0:l//2]==num[l//2+1:][::-1]

File: numerical-palindrome-number-1/numerical-palindrome-number-1.js
function palindrome(num) {
if(typeof num !== "number"||num<0){return "Not valid"}
var num = num.toString();
function pali(num){
if( num.length==2 && num[0]==num[1] || num.length==1){return true}
if( num[num.length-1]==num[0] ){return pali(num.substring(1,num.length-1))}
return false
}
return pali(num)
} 